builtin docs (#122)
kadena-io/pact-5: docs/builtins/Operators/not-q.md

## not?

> Use `not?` to apply logical 'not' to the results of applying a value to an application function.

Reviewer: ```suggestion
Use `not?` to compute logical 'NOT' of the result of applying the function `app` to the `value`.
```
